lib/cocoapods-core/specification/dsl.rb

```diff
@@ -1508,6 +1508,21 @@ def subspec(name, &block)
                 :default_value => false,
                 :spec_types => [:test]
 
+      # @!method app_host_name=(name)
```

Comment:
Should we specify that `requires_app_host` is required for this to work? is it true?

Reply:
we should, since the validator checks for it

Reply:
alternatively, should setting `app_host_name` add a dependency to that spec and also enable `requires_app_host`?

Reply:
I think it's a bad user experience to require 2 lines to be written by rote whenever you have another one, when the user's intent is clear

Reply:
i think having `requires_app_host` be auto-set makes sense but I would really rather not auto-require dependencies since that reduces visibility into what's actually happening